sdk/core/CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
 
 add_subdirectory(azure-core)
+if (NOT DISABLE_AZURE_CORE_OPENTELEMETRY)
```

Comment:
I am assuming other SDKs have this on by default. So, am I right to assume we don't want this to be opt-in, something like `ENABLE_AZURE_CORE_OPENTELEMETRY`?

Reply:
This was added when OpenTelemetry had a bug that prevented it from compiling. This allowed the CI system to make forward progress even if the opentelemetry-cpp package was broken.

Reply:
The opentelemetry-cpp vcpkg cannot compilie for UWP, so we need a mechanism to disable opentelemetry on UWP builds.

It stinks, but it's what we've got.